ansible/roles/kstest-master/files/scripts/kstests_history.py

```diff
@@ -25,6 +25,9 @@
 md5sum_filename = args.isomd5sum_filename
```

Comment:
Typo in commit b71a0283d2c954ebe52e44fcec2009434b6004f1 :
```
summary html: add test run time to header *fo* summary html page
```

Reply:
Fixed